src/components/tracks-table/TracksTable.js

```diff
@@ -120,7 +120,7 @@ const repostButtonCell = (val, record, props) => {
 }
 
 const optionsButtonCell = (val, record, index, props) => {
-  const { is_delete: deleted } = record
+  const deleted = record.is_delete || !!record.user?.is_deactivated
```

Comment:
any reason why some lines have the double bang operator while others don't? e.g. this one v. repostButtonCell

Reply:
Some places (like here) pass the result value into the `isDeleted` prop of a component, which expects a boolean (and since I user `user?` it could be undefined). I'm curious though whether the user prop is ever actually undefined/null